pkg/cli/environment.go

```diff
@@ -141,6 +144,22 @@ func New() *EnvSettings {
 	return env
 }
 
+// FillHelmSettings will fill the helm cli.Settings with the proper values from CLI flags
+// This has to be called after the flags are set, otherwise the flags are not propagated to Helm cli.Settings
+func (s *EnvSettings) FillHelmSettings() {
```

Comment:
we could use this function to substitute the assignments in `New()` now!
https://github.com/rancher-sandbox/hypper/pull/118/files#diff-ab0b247f8d73a6afc9c64237096ddb96ca5df0558fe5f5da64f493c2abad4be5R115-R125

Reply:
if we need to call this after the flags are set, I wonder if we could call FillHelmSettings() at the end of AddFlags(), and call it a day.

Reply:
We cant :cry: I already tried.

We need to wait until the flags are parsed, which happens later than that :_/

Reply:
oh! Thanks for the explanation, resolving :).

Reply:
wait. We could still using the new function in `New()`, for the assignments there, no?

Reply:
Good point! yes we can!